qemu/tests/cgroup.py

```diff
@@ -826,7 +826,7 @@ def _get_stat(f_stats, _stats=None):
 
         logging.info("Init")
         try:
-            speeds = eval(params.get('cgroup_speeds', '[10000, 100000]'))
+            speeds = eval(params.get('cgroup_speeds', '[100000, 100000, 100000]'))
```

Comment:
Is this change works for both RHEL7 and RHEL8?

Reply:
(1/1) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.8.x86_64.io-github-autotest-qemu.cgroup.cpu_share: PASS (674.39 s)

Reply:
(1/5) repeat1.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpu_share: PASS (226.19 s)
 (2/5) repeat2.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpu_share: PASS (229.50 s)
 (3/5) repeat3.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpu_share: PASS (230.54 s)
 (4/5) repeat4.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpu_share: PASS (228.99 s)
 (5/5) repeat5.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpu_share: PASS (226.52 s)

Reply:
only need fix  'float' TypeError,  don't need to fix 'cpu_share' , As all results PASS after testing multiplu times.  float TypeError will be fixed in another commit[1].  so close this commit. 
[1]https://github.com/autotest/tp-qemu/pull/1985